plugin/providers/keboola/resource_keboola_snowflake_writer_tables.go

```diff
@@ -221,13 +251,24 @@ func resourceKeboolaSnowflakeWriterTablesRead(d *schema.ResourceData, meta inter
 
 	var tables []map[string]interface{}
 
+	storageInputTableMap := make(map[string]SnowflakeWriterStorageTable)
+
+	for _, storageInputTable := range snowflakeWriter.Configuration.Storage.Input.Tables {
+			storageInputTableMap[storageInputTable.Source] = storageInputTable
+	}
+
 	for _, tableConfig := range snowflakeWriter.Configuration.Parameters.Tables {
+		storageInputTable := storageInputTableMap[tableConfig.TableID]
 		tableDetails := map[string]interface{}{
-			"db_name":     tableConfig.DatabaseName,
-			"export":      tableConfig.Export,
-			"table_id":    tableConfig.TableID,
-			"incremental": tableConfig.Incremental,
-			"primary_key": tableConfig.PrimaryKey,
+			"db_name":       	tableConfig.DatabaseName,
+			"export":        	tableConfig.Export,
+			"table_id":      	tableConfig.TableID,
+			"incremental": 	 	tableConfig.Incremental,
+			"primary_key": 		tableConfig.PrimaryKey,
+			"changed_since": 	storageInputTable.ChangedSince,
+			"where_column":		storageInputTable.WhereColumn,
+			"where_operator": 	storageInputTable.WhereOperator,
+			"where_values": 	storageInputTable.WhereValues,
```

Comment:
What happens when `storageInputTable`'s `ChangedSince`, `WhereColumn`, `WhereOperator`, and `WhereValues` fields were omitted/zero-valued during the Create? Any weird results when you apply an update?